assets/app/scripts/directives/nav.js

```diff
@@ -5,7 +5,7 @@ angular.module('openshiftConsole')
     return {
       restrict: 'E',
       templateUrl: 'views/_sidebar.html',
-      link: function($scope, element, attrs) {
+      link: function($scope) {
```

Comment:
for framework functions like this, maybe put the unused args in a comment? I like the indication of what args are being passed in:
```
link: function($scope /*, element, attrs*/) {
```